scripts/daq/main.go

```diff
@@ -41,13 +43,22 @@ func main() {
 	manager1.Start(context.Background())
 
 	uploadTimer := time.NewTimer(time.Second)
+
+	heartbeatInterval := time.NewTicker(3 * time.Second)
+	defer heartbeatInterval.Stop()
+
 	for {
 		select {
 		case <-uploadTimer.C:
 			err = telemetry.Upload()
 			if err != nil {
 				fmt.Printf("failed to upload telemetry data: %v\n", err)
 			}
+		case <-heartbeatInterval.C:
+			err = heartbeat.SendHeartbeat()
+			if err != nil {
+				logger.Error("Failed to send heartbeat", zap.Error(err))
+			}
 		}
 	}
```

Comment:
Can we name both to Interval? Feels weird to have one be uploadTimer and heartbeatInterval
```suggestion
	uploadInterval := time.NewTimer(time.Second)

	heartbeatInterval := time.NewTicker(3 * time.Second)
	defer heartbeatInterval.Stop()

	for {
		select {
		case <-uploadInterval.C:
			err = telemetry.Upload()
			if err != nil {
				fmt.Printf("failed to upload telemetry data: %v\n", err)
			}
		case <-heartbeatInterval.C:
			err = heartbeat.SendHeartbeat()
			if err != nil {
				logger.Error("Failed to send heartbeat", zap.Error(err))
			}
		}
	}

```